### DISCUSSION TITLE
[Feature request] Issue label templates

### DISCUSSION THREAD

**Discussion by seanpm2001** [community/community#60170] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Issues

### Body

It would be a great feature if issue labels were kept in a configuration file under `/.github/` so that template repositories could have their custom labels assigned automatically, rather than creating them manually for every repository. Right now, too much effort is involved for something that should be automated.
We could have a YAML file that defines the HEX color code of the label, its title, its description, and some extra rules if needed.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by spenserblack** [comment]
Not perfect, but you can write a GitHub workflow to read that file and create the issues using triggers like
```yaml
on:
  workflow_dispatch:
  push:
    branches: [main]
    paths: [".github/labels.yml"]
```

There's also [this app](https://github.com/repository-settings/app), which you might find interesting.

**Reply by seanpm2001** [reply]
Yes this would work, but I feel it should be a part of the site, and not something that has to be done through additional workflow addons or external applications.
